components/association-section/index.tsx

```diff
@@ -36,33 +32,77 @@ const AssociationNotFound: React.FC<{
         ['N°RNA', formatIntFr(uniteLegale.association.idAssociation || '')],
       ]}
     />
-  </Section>
+  </>
 );
 
-const AssociationSection: React.FC<{
+export default function AssociationSection({
+  uniteLegale,
+}: {
   uniteLegale: IAssociation;
-}> = ({ uniteLegale }) => {
+}) {
   const {
     association: { idAssociation = '', data },
   } = uniteLegale;
 
   if (!data) {
-    return <AssociationNotFound uniteLegale={uniteLegale} />;
-  }
-
-  if (isAPINotResponding(data)) {
-    if (data.errorType === 404) {
-      return <AssociationNotFound uniteLegale={uniteLegale} />;
-    }
-
+    // Data can be null if the natureJuridique is an association,
+    // but no idAssociation is provided by Insee API call.
     return (
-      <AdministrationNotResponding
-        administration={data.administration}
-        errorType={data.errorType}
-      />
+      <Section
+        title={`Répertoire National des Associations`}
+        sources={[EAdministration.MI]}
+      >
+        <AssociationNotFound uniteLegale={uniteLegale} />;
```

Comment:
There is an issue here <AssociationNotFound /> needs to be rewritten as it already contains a Section https://github.com/etalab/annuaire-entreprises-site/blob/e69d27d9f87bd3040b4674ef3c731354107699c1/components/association-section/index.tsx#L15

Reply:
My bad.

Reply:
Not anymore, I removed it in this PR : https://github.com/etalab/annuaire-entreprises-site/pull/688/files/57481e5c6bc07d13e41e7947e80b4009d6ab0591#diff-b322bb495d9629eff3b2790166611b51f0402a9edceb3768813c053eb17ecb21L42

Indeed, it is used in `notFoundInfo` of `DataSection` props which wraps it inside `Section` when displaying it.